src/FsAutoComplete.Core/CompilerServiceInterface.fs

```diff
@@ -36,7 +36,7 @@ type FSharpCompilerServiceChecker(hasAnalyzers) =
   // This is used to hold previous check results for autocompletion.
   // We can't seem to rely on the checker for previous cached versions
   let memoryCache () =
-    new MemoryCache(MemoryCacheOptions(SizeLimit = Nullable<_>(2000L)))
+    new MemoryCache(MemoryCacheOptions(SizeLimit = Nullable<_>(200L)))
```

Comment:
200 might still be too much. Someone could edit 200 files within 5 minutes and want autocompletions for them all but seems unlikely.

Reply:
nit - surface as a config option? maybe something like `FSharp.fsac.cachedTypeCheckCount`?

Reply:
Done